test_plans/oneapi_queue_empty.asciidoc

```diff
@@ -0,0 +1,76 @@
+:sectnums:
+:xrefstyle: short
+
+= Test plan for sycl extension oneapi queue empty
+
+This is a test plan for the extension adding a new API that tells whether a
+queue is empty. The extension is described in
+https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/supported/sycl_ext_oneapi_local_memory.asciidoc[sycl_ext_oneapi_local_memory.asciidoc].
+
+== Testing scope
+
+=== Device coverage
+
+All of the tests described below are performed only on the default device that
+is selected on the CTS command line.
+
+[NOTE]
+====
+Currently the extension support for OpenCL backend is limited, API 
+introduced by this extension can be called only for `in-order` queues which
+doesn’t have `discard_events` property. So the queue used for the tests should
+be created with `in-order` property if backend is OpenCL.
```

Comment:
I don't think we should do this.  The CTS should test for the correct behavior according to the spec.  It should not work around bugs in the DPC++ implementation.  I expect this will lead to failures when the test is run on the OpenCL backend, but this is OK.  The failure will correctly point out a deficiency in the DPC++ implementation.

Reply:
Got it, removed.